implants/imix/src/run.rs

```diff
@@ -19,8 +19,17 @@ pub async fn handle_main() {
     }
 
     loop {
-        let cfg = Config::default_with_imix_verison(VERSION);
-        let retry_interval = cfg.retry_interval;
+        let cfg = Config::default_with_imix_version(VERSION);
+        let retry_interval = if let Some(beacon) = cfg.info.as_ref() {
+            if let Some(transport) = &beacon.active_transport {
+                transport.interval
+            } else {
+                5
+            }
+        } else {
+            5
+        };
```

Comment:
This can be simplified

```rust
let retry_interval = cfg.info.as_ref()
    .and_then(|beacon| beacon.active_transport.as_ref())
    .map(|transport| transport.interval)
    .unwrap_or(5);
 ```
 
 Also can we make `5` a static var at the top?